tasks/generate.py

```diff
@@ -154,6 +154,16 @@ def parse_args(self, args):
         except ImportError:
             packages += ["setuptools"]
 
+    # Check if the user has requested us not to install wheel
+    if "--no-wheel" in args or os.environ.get("PIP_NO_WHEEL"):
```

Comment:
This is only for whether or not wheel gets installed as part of ``get-pip.py`` or not. The other option is ``--no-setuptools``. It's not the same thing as the flag passed to pip.